sklearn/metrics/tests/test_pairwise.py

```diff
@@ -710,11 +710,11 @@ def test_parallel_pairwise_distances_diagonal(metric, global_dtype):
     assert_allclose(np.diag(distances), 0, atol=1e-10)
 
 
-@ignore_warnings
+@pytest.mark.filterwarnings("ignore:Could not adhere to working_memory config")
 def test_pairwise_distances_chunked(global_dtype):
-    # Test the pairwise_distance helper function.
+    # Test the pairwise_distance hel"per function.
```

Comment:
```suggestion
    # Test the pairwise_distance helper function.
```

Reply:
I've no idea how it ended up there... I checked the other changes and it seems they were the only 2